### ISSUE TITLE
Can't use Numpad keys (+, -, 0) to control zoom

### ISSUE THREAD

**Issue by dogancelik** [vector-im/element-web#9533]
![Turkish keyboard layout](https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/KB_Turkey.svg/900px-KB_Turkey.svg.png)

This is the keyboard layout used in Turkey.

```
0 * - (without SHIFT key)
= ? _ (with SHIFT key)
```

- Turkish keyboard doesn't have minus and plus like in US keyboard.
- Turkish users can't use Zoom In (no PLUS key), we use Numpad to control zoom on all browsers.
- All browsers (e.g. Chrome, Firefox) support Numpad keys.
- Numpad section on Turkish keyboard has `- + 0` but it doesn't work on Riot Web.

#### Numpad
![Numpad](https://user-images.githubusercontent.com/486818/56461976-1cd56e00-63c4-11e9-83d5-12f9338e0711.jpg)

#### View Menu
![View Menu](https://user-images.githubusercontent.com/486818/56461977-1cd56e00-63c4-11e9-8132-361c6847cb4b.png)

### Version information

- **OS**: Windows 10 (64-bit)
- **Version**: Riot Web Desktop client [1.0.8](https://github.com/vector-im/riot-web/releases/tag/v1.0.8)

**Comment by lampholder** [comment]
Would https://github.com/vector-im/riot-web/issues/3160 solve the problem?

**Comment by dogancelik** [comment]
@lampholder Yes. Still, fixing keyboard shortcuts would be a plus.

**Comment by aaronraimist** [comment]
Ugh sorry https://github.com/vector-im/riot-web/pull/8381 probably broke this. Good news is Electron did implement my feature request to allow hidden items to keep working but it is only available starting in Electron 6.0.0.beta.1 but Riot is using 4.1.3 so we can't use that yet. @jryans I guess we should put back the other menu item?

https://github.com/electron/electron/pull/16853

**Comment by jryans** [comment]
> @jryans I guess we should put back the other menu item?

Bluh, yes I suppose so. 😭 (I am just sad that Electron makes this so hard...)

**Comment by aaronraimist** [comment]
> Numpad section on Turkish keyboard has - + 0 but it doesn't work on Riot Web.

First I read this issue as just zoom in (+/=) didn't work but @dogancelik are you saying zoom out (-) and reset zoom (0) don't work either?

I put back the default electron zoom in menu item in #9654 which should fix the zoom in at least but if the other ones don't work then I don't know what the problem is.

**Comment by dogancelik** [comment]
@aaronraimist

![works-doesnt-work](https://user-images.githubusercontent.com/486818/57581872-a6a9c000-74c6-11e9-8f79-a7f1a8f4ecd5.png)

**Comment by aaronraimist** [comment]
Alright :( I don’t know what the problem is then.

On my US English keyboard the numpad keys do work.

**Comment by aaronraimist** [comment]
@dogancelik https://github.com/vector-im/element-desktop/pull/202 may have fixed this. You can try Element Nightly and see if it works now. https://packages.riot.im/nightly/install/

**Comment by t3chguy** [comment]
It has for me on macos + USB keyboard, shout if still broken for you.

**Comment by dogancelik** [comment]
I downloaded the nightly, it has the same key bindings as before. Even though it shows `Ctrl + -, Ctrl + 0`, they don't work in TR keyboard layout. If I switch to US kb. layout then it works.
![zoom crop](https://user-images.githubusercontent.com/486818/120866114-f7058600-c597-11eb-9c9f-7ddf1511cc9a.png)

Here's what it looks like when I use the numpad keys.
![zoom working](https://user-images.githubusercontent.com/486818/120866419-90349c80-c598-11eb-982e-d8ea166290ad.png)

This is not a problem of Element but of Electron. I found a solution [here](https://github.com/electron/electron/issues/5256) and will submit a PR if you agree.
